enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -218,9 +239,30 @@ An example IAM Trust Policy:
 
 ```
 
+#### ExternalDNS Operator Details
+
+Prior to this enhancement, the [ExternalDNS Operator API](https://github.com/openshift/external-dns-operator/blob/main/api)
+did not allow users to configure an AWS role ARN field. As a result, there wasn't a supported way to use the External
+DNS Operator to create DNS records in another AWS account within a shared VPC.
+
+The ExternalDNS Operator configures and runs the [ExternalDNS](https://github.com/openshift/external-dns) binary.
+Conveniently, ExternalDNS already supports the [`--aws-assume-role`](https://github.com/openshift/external-dns/blob/fe00b4b83c2263282a9068655e8e3fbbc167b653/docs/faq.md#can-external-dns-manageaddremove-records-in-a-hosted-zone-which-is-setup-in-different-aws-account)
```

Comment:
I think we can mention the fact that we considered [the external-id flag](https://github.com/kubernetes-sigs/external-dns/blob/22da9f231dbc6faa3a668b507a4a06823a129609/pkg/apis/externaldns/types.go#L460) and decided not to implement for now.

Reply:
I didn't think during the first review but may be `external-id` flag can be considered as an alternative, so placed into `Alternative` section?

Reply:
Sure, that makes sense. I made it a bit more generic to apply to OCM generally and ExternalDNS and moved it to Alternatives.